project-base/config/parameters.yaml.dist

```diff
@@ -1,22 +1,17 @@
+# @deprecated All parameters will be set by ENV variables only in next major version
 parameters:
-    database_host: postgres
-    database_port: ~
-    database_name: shopsys
-    database_user: root
-    database_password: root
-
-    elasticsearch_host: elasticsearch:9200
-
-    redis_host: redis
-
-    mailer_transport: smtp
-    mailer_host: smtp-server
-    mailer_user: ~
-    mailer_password: ~
+    database_host: '%env(DATABASE_HOST)%'
+    database_port: '%env(DATABASE_PORT)%'
+    database_name: '%env(DATABASE_NAME)%'
+    database_user: '%env(DATABASE_USER)%'
+    database_password: '%env(DATABASE_PASSWORD)%'
+    elasticsearch_host: '%env(ELASTICSEARCH_HOST)%'
+    redis_host: '%env(REDIS_HOST)%'
+    mailer_transport: '%env(MAILER_TRANSPORT)%'
+    mailer_host: '%env(MAILER_HOST)%'
+    mailer_user: '%env(MAILER_USER)%'
+    mailer_password: '%env(MAILER_PASSWORD)%'
     mailer_disable_delivery: false
     mailer_master_email_address: no-reply@shopsys.com
     mailer_delivery_whitelist: [/@shopsys\.com$/]
```

Comment:
Oh, now I see that those parameters can be set by ENV. But I think that this is a little bit confusing

Reply:
Sadly there is not better option to achieve that, because of swift mailer. I am sad as a developer of SSFW by doing so. Anyway as implementator I do not need to care about it becase it works as it is described in our doc.